web/client-api/src/main/java/io/deephaven/web/client/api/WorkerConnection.java

```diff
@@ -945,9 +945,9 @@ public BrowserHeaders metadata() {
             // table's creation, which can race this
             BiDiStream<FlightData, FlightData> stream = this.<FlightData, FlightData>streamFactory().create(
                     headers -> flightServiceClient.doPut(headers),
-                    (firstPayload, headers) -> browserFlightServiceClient.openDoPut(firstPayload, headers),
-                    (nextPayload, headers, callback) -> browserFlightServiceClient.nextDoPut(nextPayload, headers,
-                            callback::apply));
+                    (first, headers) -> browserFlightServiceClient.openDoPut(first, headers),
+                    (next, headers, callback) -> browserFlightServiceClient.nextDoPut(next, headers, callback::apply),
```

Comment:
Minor inconsistency with later, could just call `callback` `c` instead

Reply:
`c` is already defined in line 891, can't declare it again